### PR TITLE
Fixed broken link in _docker.py

### PR DESCRIPTION
**Pull request by ArnabRaxit**
##### SUMMARY
There is a broken link in this page: https://docs.ansible.com/ansible/2.6/modules/docker_module.html
This PR fixes the broken link.
The broken link was: https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/guide_docker.rst.
Relaced that with: https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION
Based on discussions on #50851 . As recommended by @mattclay  and @acozine.

### CODE REVIEW

**Review by mattclay — APPROVED** [review]

File: lib/ansible/modules/cloud/docker/_docker.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 description:
   - This is the original Ansible module for managing the Docker container life cycle.
   - NOTE - Additional and newer modules are available. For the latest on orchestrating containers with Ansible
-    visit our Getting Started with Docker Guide at U(https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/guide_docker.rst).
+    visit our L(Getting Started with Docker Guide,.../scenario_guides/guide_docker.html).
```
Comment (felixfontein) [inline lib/ansible/modules/cloud/docker/_docker.py]: I think there's a dot too many:
```suggestion
    visit our L(Getting Started with Docker Guide,../scenario_guides/guide_docker.html).
```

Comment (ArnabRaxit) [reply lib/ansible/modules/cloud/docker/_docker.py]: I just built the webdocs locally and tested it. two dots is right. Thnx @felixfontein

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by abadger** [comment]
Merged for the 2.6.13 release.

**Comment by felixfontein** [comment]
@ArnabRaxit thanks for your PR! Do you want to do the backport to stable-2.5 yourself, or should I do it?
@mattclay thanks for reviewing!
@abadger thanks for merging!

**Comment by abadger** [comment]
I'm not sure we would merge a backport to 2.5
  It is docs only but inside of a module file.  @nitzmahone @acozine care to weigh in before any work is done here?

**Comment by felixfontein** [comment]
The original PR (#50851) was for stable-2.5, so it looks like that people still stumble over this in the 2.5 docs. Anyway, let's see what @nitzmahone and @acozine think.

**Comment by acozine** [comment]
I see this as a bugfix - the link is broken. So I'm fine with backporting it to all currently-supported versions. We've got several months of support for 2.5 ahead of us.

**Comment by nitzmahone** [comment]
Tis OK by me- esp since it doesn't require a release to get the fix out to the majority of the world.

**Pull request merged**